Jenkinsfile

```diff
@@ -31,13 +31,13 @@ node('linux && docker') {
         junit 'packages/*/reports/*.xml'
       }
 
-      //stage('Cypress Test') {
-      //  sh 'apt-get -y install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libgbm-dev libnss3 libasound2 xauth xvfb'
-      //  sh 'rm -rf /var/lib/apt/lists/*'
-      //  sh 'cd packages/atomic && npx cypress install'
-      //  sh 'npm start & npx wait-on http://localhost:3333'
-      //  sh 'NO_COLOR=1 npm run cypress:test'
-      //}
+      stage('Cypress Test') {
+       sh 'apt-get -y install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libgbm-dev libnss3 libasound2 xauth xvfb'
```

Comment:
Ignore this if it was already asked & answered, but where is it defined that those packages are needed? Would there be a way not to have to manually specify them?

Reply:
Yes, the question was asked when I first added this into jenkinfile  a while ago.  I cant recall where I got the list of needed package, probable https://github.com/cypress-io/github-action/issues/251 + ref with commerce team. 
Alexis did recommend to create our Docker image and upload to S3 to reuse, but we haven't really checked that